cmd/vulcan-vulners/main.go

```diff
@@ -105,8 +105,17 @@ func severity(score float32) string {
 	}
 }
 
-// vulnerabilities retrieves the vulnerabilities affecting a software component from the vulners.com API.
-func vulnerabilities(p, s, v, t string) (*report.Vulnerability, error) {
+type vulnerFinding struct {
+	Score    float32
+	Resource report.ResourcesGroup
```

Comment:
More OCD, but I'd put `Resources` in plural because it's a group of resources.

Reply:
Okey makes sense